test/recipient_test.rb

```diff
@@ -0,0 +1,35 @@
+require_relative "test_helper"
```

Comment:
Generally you're never instantiating `Recipient`, it makes more sense as an abstract class which you would test through `Channel` and `User`.  

Also I suggest trying to test with minimal dependencies, so testing Recipient without having to use Workspace.